### ISSUE TITLE
dependency problems prevent configuration of kali-anonsurf

### ISSUE THREAD

**Issue by seventhplug** [Und3rf10w/kali-anonsurf#15]
#./installer.sh 
--2016-11-22 20:06:01--  https://geti2p.net/_static/i2p-debian-repo.key.asc
Resolving geti2p.net (geti2p.net)... 91.143.92.136, 2a02:180:a:65:2456:6542:1101:1010
Connecting to geti2p.net (geti2p.net)|91.143.92.136|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 14455 (14K) [text/plain]
Saving to: ‘/tmp/i2p-debian-repo.key.asc’

/tmp/i2p-debian-rep 100%[===================>]  14.12K  7.57KB/s    in 1.9s    

2016-11-22 20:06:11 (7.57 KB/s) - ‘/tmp/i2p-debian-repo.key.asc’ saved [14455/14455]

OK
Hit:1 http://ftp.yzu.edu.tw/Linux/kali kali-rolling InRelease                  
Hit:2 http://deb.i2p2.no unstable InRelease        
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
i2p-keyring is already the newest version (2016.10.01).
The following packages were automatically installed and are no longer required:
  libpoppler61 python-brotli secure-delete
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 25 not upgraded.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
secure-delete is already the newest version (3.1-6).
secure-delete set to manually installed.
tor is already the newest version (0.2.8.9-1).
tor set to manually installed.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 i2p : Depends: i2p-router (>= 0.8.6-5) but it is not going to be installed
       Depends: libjbigi-jni but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
dpkg-deb: building package 'kali-anonsurf' in 'kali-anonsurf.deb'.
Selecting previously unselected package kali-anonsurf.
(Reading database ... 310708 files and directories currently installed.)
Preparing to unpack kali-anonsurf.deb ...
Unpacking kali-anonsurf (1.2.2) ...
dpkg: dependency problems prevent configuration of kali-anonsurf:
 kali-anonsurf depends on i2p; however:
  Package i2p is not installed.

dpkg: error processing package kali-anonsurf (--install):
 dependency problems - leaving unconfigured
Processing triggers for systemd (232-3) ...
Errors were encountered while processing:
 kali-anonsurf
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following packages were automatically installed and are no longer required:
  libpoppler61 libservice-wrapper-java libservice-wrapper-jni python-brotli
  secure-delete service-wrapper
Use 'apt autoremove' to remove them.
The following packages will be REMOVED:
  kali-anonsurf
0 upgraded, 0 newly installed, 1 to remove and 25 not upgraded.
1 not fully installed or removed.
After this operation, 65.5 kB disk space will be freed.
Do you want to continue? [Y/n] 

┌─[root@Silver]─[~/Downloads/kali-anonsurf]
└──╼ #uname -a
Linux Silver 4.8.0-kali1-amd64 #1 SMP Debian 4.8.5-1kali1 (2016-11-04) x86_64 GNU/Linux
┌─[root@Silver]─[~/Downloads/kali-anonsurf]
└──╼ #cat /etc/os-release 
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2016.2"
VERSION_ID="2016.2"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.kali.org/"
SUPPORT_URL="http://forums.kali.org/"
BUG_REPORT_URL="http://bugs.kali.org/"
┌─[root@Silver]─[~/Downloads/kali-anonsurf]
└──╼ #

**Comment by Und3rf10w** [comment]
This appears to be a duplicate of #13 

I have no idea why you're having issues attempting to install ```libjbigi-jni```, that installed just fine on my Kali instance.

Per #13, please attempt the following:

```bash
apt-get update
apt-get install libservlet3.0-java #if this fails, I suggest you start from a fresh Kali install
wget http://ftp.us.debian.org/debian/pool/main/j/jetty8/libjetty8-java_8.1.16-4_all.deb
dpkg -i libjetty8-java_8.1.16-4_all.deb # This should succeed without error
apt-get install libecj-java libgetopt-java libservlet3.0-java glassfish-javaee ttf-dejavu i2p i2p-router libjbigi-jni #installs i2p and other dependencies
apt-get -f install # resolves anything else in a broken state
```

**Comment by Und3rf10w** [comment]
Fixed by 704c453
